apps/web/src/server/applications/case/representations/representation-details/application-representation-details.view-model.js

```diff
@@ -0,0 +1,151 @@
+import { format } from 'date-fns';
+
+/**
+ * @typedef {object} Representation
+ * @property {number} id
+ * @property {string} reference
+ * @property {string} status
+ * @property {Contact[]} contacts
+ * @property {boolean} redacted
+ * @property {string} received
+ * @property {string} originalRepresentation
+ * @property {string|null} redactedRepresentation
+ */
+
+/**
+ * @typedef {object} Address
+ * @property {string|null} addressLine1
+ * @property {string|null} addressLine2
+ * @property {string|null} town
+ * @property {string|null} county
+ * @property {string|null} postcode
+ */
+
+/**
+ * @typedef {object} Contact
+ * @property {string|null} firstName
+ * @property {string|null} lastName
+ * @property {string|null} organisationName
+ * @property {string} type
+ * @property {string|null} jobTitle
+ * @property {boolean} under18
+ * @property {string|null} email
+ * @property {string|null} phoneNumber
+ * @property {Address} address
+ */
+
+/**
+ * @param {object} representation
+ * @param {Contact[]} representation.contacts
+ * @param {string} type
+ * @returns {object}
+ */
+const getContactDetailsByContactType = ({ contacts }, type) => {
+	let res = {};
```

Comment:
use response if following  the response object pattern. 

Mainly in express to ignore the mistake of thinkings it's req/res